Sources/ArcGISToolkit/Components/Offline/Utilities/URL+Offline.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Foundation
+
+extension URL {
+    /// The path to the web map directory for a specific portal item.
+    /// `Documents/OfflineMapAreas/<Portal Item ID>/`
+    /// - Parameter portalItemID: The ID of the web map portal item.
+    static private func portalItemDirectory(forPortalItemID portalItemID: PortalItem.ID) -> URL {
+        URL.documentsDirectory.appending(components: "OfflineMapAreas", "\(portalItemID)/")
```

Comment:
Explicit `URL` isn't needed.

```suggestion
        return .documentsDirectory.appending(components: "OfflineMapAreas", "\(portalItemID)/")
```